blueprint/office-365.md

```diff
@@ -1233,27 +1233,35 @@ Alert Policies Status | Default Alert Policies enabled | Custom alert policies a
 
 * Manually - The label is applied manually by the end-user.
 * Automatically applied based on the location of the document - Labels can be configured to automatically apply based on the location of the document. For example, SharePoint.
 * Automatically applied based on detected Sensitive Information Type - Labels can be configured to automatically apply based on the type of sensitive information found. For example, documents containing Australia driver's license numbers.
 
-At the time of writing, Sensitivity labels cannot be configured to satisfy some specific requirements listed in the Protective Security Policy Framework (PSPF). The PSPF requires the protective marking to be applied to email messages via either:
+At the time of writing, Sensitivity labels cannot be configured to satisfy some specific requirements listed in the Protective Security Policy Framework (PSPF). Some of these limitations are as follows:
 
-* Appending the protective marking to the Subject field using a specified syntax (Subject Field Marking).
-* Including the protective marking in an Internet Message Header Extension using a specified syntax (Internet Message Header Extension).
+- Sensitivity labels do not set the `X-Protective-Marking` header (internet message header extension).
+- Messages received from outside of the Agency that have the appropriate  `X-Protective-Marking ` header set are not interpreted by the native labelling in the Outlook client.
+- Sensitivity labels are not available within a shared or delegated mailbox in the iOS Outlook client.
+- The MIP labelling clients cannot append the protective marking to the subject field (subject field marking) to an email.
+- When downgrading a Sensitivity label, the downgrade cannot be prevented, only forcing the user to justify the downgrade. 
+- Sensitivity labels are not available within calendar invites.
 
-Sensitivity labels create a protective marking within the message header and when combined with Exchange mail flow rules the subject can be modified to prepend text in the subject according to the sensitivity.
+The following methods can be used to assist with PSPF compliance when using MIP:
 
-Email gateway rules are available in the Network Configuration ABAC document. These rules are based on regular expressions and are easily adaptable to vendor specific email gateways.
+* Prepend the protective marking to the subject field using Exchange Online transport rules and MIP header checks.
+* For calendar invites, advise users to add the classification to the body, and use transport rules to add the required subject and header values.
+* Add the  `X-Protective-Marking` header associated with the MIP label at the email gateway, as transport rules cannot not populate the user's email address in the `ORIGIN=user@agency.gov.au` property inside the `X-Protective-Marking` header.
```

Comment:
`as transport rules cannot not populate`

Is `cannot not` correct?

Reply:
fixed